### DISCUSSION TITLE
Any way to set up 2 effect queue for 2 output virtual/devices at once?

### DISCUSSION THREAD

**Discussion by goremil** [wwmm/easyeffects#1387] [Q&A]
Hi.
Is it possible to set-up easyeffect with active 2 separate effect queue for 2 different outputs?

Example:
```
1st app (eg. Media Player) >>> 1st eassyeffect queue >>> 1st dev/sink (eg. Speakers)
2nd app (eg. Voice Comm)   >>> 2nd eassyeffect queue >>> 2nd dev/sink (eg. Headphones)
```

**Comment by wwmm** [comment]
There is some discussion about this on #1145. We are talking about a really deep change to the application. I doubt it is going to happen some day.

**Comment by goremil** [comment]
Thank you for answer.
I hope that functionality will come some day to easyeffect.

As I researching now, it is possible by patchbay for JACK like Carla - but that is different story - complicated pro-audio :(

**Reply by wwmm** [reply]
It would be nice. But it is so hard to do... Besides all the rework that has to be done under the hoods there is the challenging of implementing a graphical interface where "per stream effects pipelines" can exist without removing the current behavior that is having only one pipeline for all streams.
